README.md

```diff
@@ -89,4 +89,24 @@ rows = [
 ]
 data = [get_avro_row_dict(row, schema) for row in rows]
 
-```
\ No newline at end of file
+```
+
+### Overriding mappings
+
+Some cases might require overriding standard mapping. To do so, simply pass your mapping overrides as a column name keyed dict to the `get_avro_schema` function like so:
+
```

Comment:
I would mention here the main reason we're doing this charade - numeric types.

Reply:
Explanation and example added